cypress/commons/actions/generic/TableParameters.js

```diff
@@ -164,7 +176,14 @@ function clearStringCell(getTableElement, colName, rowIndex) {
   return getCell(getTableElement(), colName, rowIndex);
 }
 
+function scrollTable(getTableElement, position) {
```

Comment:
Could we rename this function to `scrollTo` or `scrollToPosition`?

Also, could you add a comment to explain what value is expected for the `position` argument? When reading this code, I have absolutely no idea if this should be a position in pixels, an integer for a row number, a DOM element selected by cypress, or something else... :sweat_smile:

Reply:
We do not need it with new test